test/blackbox-tests/test-cases/env-cflags/dune-project

```diff
@@ -0,0 +1 @@
+(lang dune 1.0)
```

Comment:
This will need to be 1.7 as you add the appropriate Syntax.since

Reply:
indeed, it does, which means I added Syntax.since correctly. fixed.